### ISSUE TITLE
Error in configuration example

### ISSUE THREAD

**Issue by Silvanoc** [containers/storage#196]
Current configuration example provides the [field `ostree_repo` under section `[storage.options.thinpool]`](https://github.com/containers/storage/blob/90d0a58ffabb324042e074d33d1f6fbf3185c789/storage.conf#L125), but it has to be in section `[storage.options]` to work.

**Comment by Silvanoc** [comment]
Running `containers-storage` with the provided configuration results in following output:
```
Root: /var/lib/containers/storage
Run Root: /var/run/containers/storage
Driver Name: overlay
Driver Options: [overlay2.override_kernel_check=false]
Backing Filesystem: extfs
Supports d_type: true
Native Overlay Diff: true
```

Moving the `ostree_repo` value to the section `[storage.options]` results in following output:
```
Root: /var/lib/containers/storage
Run Root: /var/run/containers/storage
Driver Name: overlay
Driver Options: [overlay2.ostree_repo=/var/lib/containers/ostree-repo overlay2.override_kernel_check=false]
Backing Filesystem: extfs
Supports d_type: true
Native Overlay Diff: true
```
which clearly reports the usage of OStree on Overlay2.

**Comment by Silvanoc** [comment]
Fixed with commit 9f0144d

**Comment by Silvanoc** [comment]
I just got my company clearance and wanted to submit a PR fixing it, to find out that @giuseppe already had :smile:

Next time I'll submit a PR fixing it, since I now have the clearance from my company to do so.
